README.md

```diff
@@ -64,76 +46,44 @@ producing complex, multi-container applications.
         # run following command to configure your shell:
```

Comment:
Can you provide those for @Preeticp  - her machine got borked so she may have a delay before she can run a test run